packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -66,6 +66,14 @@ export const SignInFactorOnePasswordCard = (props: SignInFactorOnePasswordProps)
           gap={8}
         >
           <Form.Root onSubmit={handlePasswordSubmit}>
+            {/* For password managers */}
+            <input
+              readOnly
```

Comment:
❓Why did we decided not to go with type='hidden' ? Do we have extra benefits with this approach ?